TTHAnalysis/python/plotter/ttH-multilepton/ttH_plots.py

```diff
@@ -5,44 +5,60 @@
 
 ODIR=sys.argv[1]
 YEAR=sys.argv[2]
+Tag =sys.argv[3]
 lumis = {
 '2016': '35.9',
 '2017': '41.4',
 '2018': '59.7',
 'all' : '35.9,41.4,59.7',
 }
 
-
 submit = '{command}' 
 dowhat = "plots" 
 #dowhat = "dumps" 
 #dowhat = "yields" 
 #dowhat = "ntuple" # syntax: python ttH-multilepton/ttH_plots.py no 2lss_SR_extr outfile_{cname}.root --sP var1,var2,...
-
-P0="/eos/cms/store/cmst3/group/tthlep/peruzzi/"
```

Comment:
Wrong, you are breaking the paths for everyone. Please maintain my version in which we conditionally use our paths if os.environ['HOSTNAME'] contains  cism.ucl.ac.be

Reply:
DONE